bloom/generators/debian/templates/ament_python/copyright.em

```diff
@@ -1,11 +1,11 @@
-Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Format: Bloom subset of https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Name: @(Name)
 Upstream-Contact: @(Maintainers.replace(', ', '\n '))
 @[if Source and Source != '']Source: @(Source)@\n@[end if]@
 @[for License, Text in Licenses]@
 
-Files: *
+Files: package.xml
```

Comment:
I still think it would be best if the placeholder value here were not a valid source file since to avoid generating an inaccurate file. Going back to the vendor package examples with license tags for the vendor package and the vendored package, the package.xml file itself would explicitly be Apache licensed _and not_ be Boost licensed versus appearing for both.